run-wsgi-dev

```diff
@@ -23,4 +23,4 @@ export CERT_PATH=test/cert/cert.crt
 export KEY_PATH=test/cert/private.key
 export API_RESPONSE_KEY=xS5MqJ6N9CyH-hvqAGrmBVAxFMOyauMpdrdqCZa1eqo=
 export IP_AND_PORT=$(hostname -I | awk '{print $1}'):5000
-./venv/bin/uwsgi --http 0.0.0.0:5000 --gevent 100 --wsgi-file wsgi.py --callable app
+./venv/bin/uwsgi --http 0.0.0.0:5001 --gevent 100 --wsgi-file wsgi.py --callable app
```

Comment:
It seems like in the docker environment a python venv is not needed.  So changing this line to `uwsgi --http 0.0.0.0:5001 --gevent 100 --wsgi-file wsgi.py --callable app` seems to have it find uwsgi.  pip installs it someplace in path luckily!